### ISSUE TITLE
Implement VFS Team Offboarding process

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#90155]
## Status
| Date | Status | Launch Date | Notes |
| ----- | ------ | ------------------------- | ------ |
| 8/26/2025 | In progress |8/12/2025 | Product outline complete. Closing initiative. Governance and Platform Support will demo this at the EOS demo on 8/27.|
| 8/22/2025 | In progress |8/12/2025 | Product outline updates are in progress.|
| 8/15/2025 | In progress |8/12/2025 | We are now live and have launched. On track to close out end of next sprint once Product Outline is complete.|
| 8/1/2025 | In progress | | A couple minor changes needed. On track to launch 8/11.|
| 7/25/2025 | In progress | | Updated Platform Website. On track to launch next sprint.|
| 6/27/2025 | In progress | | Finalizing updates to operational guidance. Will work on updates to Platform Website next.|
| 6/20/2025 | In progress | | Drafting operational guidance. Checking in with other Platform teams. Updating current Onboarding to "Individual Onboarding"|
| 6/13/2025 | In progress | | Design of new ticket has been drafted. Will connect with Platform Security and other teams next sprint to finalize.|
| 6/6/2025 | In progress | | Continuing to draft team and product templates, and new offboarding ticket|
| 5/30/2025 | In progress | | Editing flow based on Platform Support feedback. Formalizing on Monday 6/2|
| 5/23/2025 | In progress | | Creating draft templates and connecting with Platform Support.|
| 5/16/2025 | In progress | | Mock-up finalized. Reviewing and will share with PO for approval.|
| 5/12/2025 | In progress | | Mock-up/design plan in progress. |
| 5/1/2025 | In progress | | Synthesis complete. Solutioning session scheduled for 5/5 |
| 4/18/2025 | In progress | | Synthesis still in progress. |
| 4/10/2025 | In progress | | Research plan documented, research completed. Synthesis in progress. |
| 3/7/2025 | Not started | | On track to begin next sprint, 3/13/2025 |
| 2/24/2025 | Not started | | Discussing in refinement. Planning to pick up mid-March |
| 1/13/2025 | Not started | | Will begin planning soon. Expected to start before end of CY25Q1 |
| 12/16/2024 | Not started | | Concerns whether scope if full responsibility of Governance team |

## Problem Statement 
Platform would like to introduce a new process for Offboarding VFS teams when their contracts end. While the current individual Offboarding process ensures that individuals' access to Platform products and services is removed when they exit their contract, a team offboarding process would help ensure OCTO-DE is aware of which products are no longer being maintained and/or supported for VA.gov. In addition, a team offboarding process will also ensure that appropriate escalation occurs if/when there are any identified risks due to a lapse in service/maintenance. 

## Hypothesis or Bet
If we implement a VFS team offboarding process, we'll be able to identify any risks or potential risks to OCTO and/or VA.gov Platform that could impact service to Veterans.

If we implement a VFS team offboarding process, we'll be able to identify orphaned products to communicate to OCTO-DE for action.

If we implement a VFS team offboarding process, our points of contact in Atlas and other tools will remain up to date.

## We will know we're done when... ("Definition of Done")

- [ ] We have completed research with VA.gov Platform teams and OCTO leads to determine what concerns them if/when teams offboard.
- [ ] We have prioritized the top concerns from our research to be incorporated into the team offboarding process.
- [ ] We have defined and published a new Team Offboarding process.
- [ ] We have announced and communicated the new Team Offboarding process to the VA.gov audience, including VFS teams.

## Known Blockers/Dependencies 
N/A

## Projected Launch Date
* When do you expect to be completed rolling out this initiative*

### Guidance (delete before posting)

_This checklist is intended to be used to help answer, "is my Platform initiative ready for launch?". All of the items in this checklist should be completed, with artifacts linked---or have a brief explanation of why they've been skipped---before launching a given Platform initiative. All links or explanations can be provided in **Required Artifacts** sections. The items that can be skipped are marked as such._

_Keep in mind the distinction between **Product** and **Initiative** --- each Product needs specific supporting documentation, but Initiatives to improve existing Products should reuse existing documentation for that Product.  [VSP Product Terminology](https://vfs.atlassian.net/wiki/spaces/PMCP/pages/1912569976/Platform+Product+terminology?atlOrigin=eyJpIjoiM2VmNzY2MTZkZWE4NGY2NmJmNjk2YWIwMmJhMGZiZmYiLCJwIjoiY29uZmx1ZW5jZS1jaGF0cy1pbnQifQ) for details._

### ... tested?

- [ ] Usability test (_TODO: link_) has been performed, to validate that new changes enable users to do what was intended and that these changes don't worsen quality elsewhere. If usability test isn't relevant for this change, document the reason for skipping it.
    - [ ] ... and issues discovered in usability testing have been addressed.
    * _Note on skipping: metrics that show the impact of before/after can be a substitute for usability testing._
- [ ] End-to-end [manual QA](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/README.md) or [UAT](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/planning/what-is-uat.md) is complete, to validate there are no high-severity issues before launching
- [ ] _(if applicable)_ New functionality has thorough, automated tests running in CI/CD

### ... documented?

- [ ] New documentation is written pursuant to our [documentation style guide](https://vfs.atlassian.net/wiki/spaces/AP/pages/622264362/Style+guide)
- [ ] Product is included in the [List of VSP Products](https://docs.google.com/spreadsheets/d/1Fn2lD419WE3sTZJtN2Ensrjqaz0jH3WvLaBtn812Wjo/edit#gid=0)
    * _List the existing product that this initiative fits within, or add a new product to this list._
- [ ] Internal-facing: there's a [Product Outline](https://vfs.atlassian.net/wiki/spaces/PMCP/pages/1924628490/Product+Outline+Template)
- [ ] External-facing: a [User Guide on Platform Website](https://vfs.atlassian.net/wiki/spaces/AP/pages/1477017691/Platform+website+guidelines) exists for this product/feature tool
- [ ] _(if applicable)_ Post to [#vsp-service-design](https://dsva.slack.com/channels/vsp-service-design) for external communication about this change (e.g. customer-facing meetings)

### ... measurable

- [ ] _(if applicable)_ This change has clearly-defined success metrics, with instrumentation of those analytics where possible, or a reason documented for skipping it.
    * For help, see: [Analytics team](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Analytics-customer-support-guide.1586823275.html)
- [ ] This change has an accompanying [Platform Release Plan](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose).

### When you're ready to launch...
- [ ] Configure your communication to teams about your work using the [Change Communication Guide](https://vfs.atlassian.net/wiki/spaces/PMCP/pages/2317647873/Change+Communication+Guide)
- [ ] Conduct a [go/no-go](https://vfs.atlassian.net/wiki/spaces/AP/pages/1670938648/Platform+Crew+Office+Hours#Go%2FNo-Go) when you're almost ready to launch. 

### Documentation
* **`PRODUCT_NAME`**: _directory name used for your product documentation_
* **Product Outline**: _link to Product Outline_
* **User Guide**: _link to User Guide_

### Testing

* **Usability test**: _link to GitHub issue, or provide reason for skipping_
* **Manual QA**: _link to GitHub issue or documented results_
* **Automated tests**: _link to tests, or "N/A"_

### Measurement

* **Success metrics**: _link to where success metrics are measured, or where they're defined (Product Outline is OK), or provide reason for skipping_
* **Release plan**: _link to Release Plan ticket_

## TODOs
- [ ] Convert this issue to an epic
- [ ] Add your team's label to this epic
- [ ] Add this initiative to the [Digital Experience Product Board](https://github.com/orgs/department-of-veterans-affairs/projects/306?card_filter_query=label%3Acrew-platform) using [these instructions](https://vfs.atlassian.net/wiki/spaces/AP/pages/2120974387/Platform+Initiative+Guidelines+for+Digital+Experience+Product+Board)

**Comment by shiragoodman** [comment]
FYSA @humancompanion-usds. This initiative likely needs refinement from various Platform teams.

**Comment by shiragoodman** [comment]
Refinement 2/24:

- We don't have a lot of visibility for teams outside of the Collaboration Cycle
- Feels a lot like PO Sync - OCTO asking us to solve a problem that could be solved internally
- Specific things we'd need to know:
   -  labeling/identifying products or features if/when the team is no longer around
   - teams that are offboarding may not actually be offboarding (if they win recompete)
   - worth asking teams about Collab Cycle status
   - Should domo dashboards be maintained?
- need to reach out to other teams to understand what they need.
- talk to our OCTO leads to determine what they need.
- what kind of tracking is OCTO doing and what are they already aware of?
- determine where this will live
- If similar to individual offboarding, need to decide _how_ to implement. Should we align it to Platform Orientation?
- Update Platform Website.
- Determine who would be responsible for submitting this Team Offboarding ticket
- Determine _when_ would we ask teams to complete the Team Offboarding ticket
- how would we prompt VFS teams to tell them that they need this
- how do we market Team offboarding? How can we ensure VFS teams and others are aware of this process?
- consider prioritization of offboarding items if we receive too many from Platform crew
- who's responsible for cleaning up whatever is left behind?
- Does VA do any "exit interviews" at the contract level? Is there an opportunity for us to tie into that?
